### ISSUE TITLE
Value of type 'SwiftFlutterAudioOutputPlugin' has no member 'channel'

### ISSUE THREAD

**Issue by ChiragLiveu** [itsmurphy/flutter_audio_output#2]
I have faced the same issue as described in title 
Value of type 'SwiftFlutterAudioOutputPlugin' has no member 'channel'
Can you please some one help to resolve that issue.

**Comment by ChiragLiveu** [comment]
I have also tried deleting and reinstalling pods. and it's successfully installed. Stil I am facing the same issue

**Comment by tmratwork** [comment]
I received same error.

**Comment by mred0** [comment]
Yes I Am Also Facing This Issue, I Think Problem In Import Of It There Some Import Missing.

**Comment by enyor** [comment]
Add line 6. It's Temporally

![Captura de Pantalla 2024-06-13 a la(s) 8 04 02 a m](https://github.com/0x01-itsmurphy/flutter_audio_output/assets/16519398/c9fc7e14-f1a0-4837-ab2b-8bb885dc21a8)

**Comment by 0x01-itsmurphy** [comment]
Hey @enyor, thanks for reporting this!

You are completely right on the fix, but just as a heads-up: adding var channel: FlutterMethodChannel? is actually the permanent and officially correct way to store the channel so native background events can properly trigger Dart methods.

This was officially added and fixed in the repository a while ago, and all modern versions of the package use this structure. I'll go ahead and close this issue. Thanks again!
